modulelogger/module_logger.go

```diff
@@ -0,0 +1,278 @@
+package modulelogger
+
+import (
+	"fmt"
+	log "github.com/uber-common/bark"
+)
+
+type Level uint8
```

Comment:
Can you split `Level` `String()` `ParseLevel()` out into a `logging/level.go` file?